sct.py

```diff
@@ -70,7 +68,8 @@
     list_logs_by_test_id,
     list_resources_docker,
     search_test_id_in_latest,
-    list_parallel_timelines_report_urls
+    list_parallel_timelines_report_urls, get_ami_images, get_gce_images,
```

Comment:
aren't these out of order? like, isn't it better to have them in alphabetical order, and one command by line?

Reply:
Not sure why PyCharm decided that's the way to place the imports. Either way, I fixed them back to alphabetical order.